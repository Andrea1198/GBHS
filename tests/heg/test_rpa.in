
   &CONTROL
  calculation = "RPA"
       rs_min =  1.0
       rs_max = 10.0
          nrs = 10
   /

   &RPA
           nq = 8000
           nw = 16000
       qshift = 0.00000
       qscale = 0.05
       wscale = 0.05

   /
