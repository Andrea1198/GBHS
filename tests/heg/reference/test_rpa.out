
  ======================================================================
  =  Input Summary                                                     =
  ======================================================================


     <CONTROL>
             calc : RPA
           rs_min :     1.000000
           rs_max :    10.000000
              nrs :    10

     <RPA>
               nq :  8000
               nw : 16000
           qshift :     0.000000
           qscale :     0.050000
           wscale :     0.050000


     Threads/MPI process:           6

  ======================================================================
  =  RPA calc                                                          =
  ======================================================================


     rs =     1.000000000
     kf =     1.919158293

     qgrid:      [ndim = 8000 ]
               0.000009816       0.000019633       0.000029449       0.000039265       0.000049081
              50.935931627      63.669921896      84.893236829     127.339863424     254.679736663

     wgrid:      [ndim =16000 ]
               0.000004908       0.000009817       0.000014725       0.000019634       0.000024542
             101.865521596     127.331905676     169.775878052     254.663821169     509.327647246


     E_corr =    -0.157799962 [Ry] 


     rs =     2.000000000
     kf =     0.959579146

     qgrid:      [ndim = 8000 ]
               0.000009816       0.000019633       0.000029449       0.000039265       0.000049081
              50.935931627      63.669921896      84.893236829     127.339863424     254.679736663

     wgrid:      [ndim =16000 ]
               0.000004908       0.000009817       0.000014725       0.000019634       0.000024542
             101.865521596     127.331905676     169.775878052     254.663821169     509.327647246


     E_corr =    -0.123810815 [Ry] 


     rs =     3.000000000
     kf =     0.639719431

     qgrid:      [ndim = 8000 ]
               0.000009816       0.000019633       0.000029449       0.000039265       0.000049081
              50.935931627      63.669921896      84.893236829     127.339863424     254.679736663

     wgrid:      [ndim =16000 ]
               0.000004908       0.000009817       0.000014725       0.000019634       0.000024542
             101.865521596     127.331905676     169.775878052     254.663821169     509.327647246


     E_corr =    -0.105726182 [Ry] 


     rs =     4.000000000
     kf =     0.479789573

     qgrid:      [ndim = 8000 ]
               0.000009816       0.000019633       0.000029449       0.000039265       0.000049081
              50.935931627      63.669921896      84.893236829     127.339863424     254.679736663

     wgrid:      [ndim =16000 ]
               0.000004908       0.000009817       0.000014725       0.000019634       0.000024542
             101.865521596     127.331905676     169.775878052     254.663821169     509.327647246


     E_corr =    -0.093823862 [Ry] 


     rs =     5.000000000
     kf =     0.383831659

     qgrid:      [ndim = 8000 ]
               0.000009816       0.000019633       0.000029449       0.000039265       0.000049081
              50.935931627      63.669921896      84.893236829     127.339863424     254.679736663

     wgrid:      [ndim =16000 ]
               0.000004908       0.000009817       0.000014725       0.000019634       0.000024542
             101.865521596     127.331905676     169.775878052     254.663821169     509.327647246


     E_corr =    -0.085136010 [Ry] 


     rs =     6.000000000
     kf =     0.319859715

     qgrid:      [ndim = 8000 ]
               0.000009816       0.000019633       0.000029449       0.000039265       0.000049081
              50.935931627      63.669921896      84.893236829     127.339863424     254.679736663

     wgrid:      [ndim =16000 ]
               0.000004908       0.000009817       0.000014725       0.000019634       0.000024542
             101.865521596     127.331905676     169.775878052     254.663821169     509.327647246


     E_corr =    -0.078445484 [Ry] 


     rs =     7.000000000
     kf =     0.274165470

     qgrid:      [ndim = 8000 ]
               0.000009816       0.000019633       0.000029449       0.000039265       0.000049081
              50.935931627      63.669921896      84.893236829     127.339863424     254.679736663

     wgrid:      [ndim =16000 ]
               0.000004908       0.000009817       0.000014725       0.000019634       0.000024542
             101.865521596     127.331905676     169.775878052     254.663821169     509.327647246


     E_corr =    -0.073054038 [Ry] 


     rs =     8.000000000
     kf =     0.239894787

     qgrid:      [ndim = 8000 ]
               0.000009816       0.000019633       0.000029449       0.000039265       0.000049081
              50.935931627      63.669921896      84.893236829     127.339863424     254.679736663

     wgrid:      [ndim =16000 ]
               0.000004908       0.000009817       0.000014725       0.000019634       0.000024542
             101.865521596     127.331905676     169.775878052     254.663821169     509.327647246


     E_corr =    -0.068581549 [Ry] 


     rs =     9.000000000
     kf =     0.213239810

     qgrid:      [ndim = 8000 ]
               0.000009816       0.000019633       0.000029449       0.000039265       0.000049081
              50.935931627      63.669921896      84.893236829     127.339863424     254.679736663

     wgrid:      [ndim =16000 ]
               0.000004908       0.000009817       0.000014725       0.000019634       0.000024542
             101.865521596     127.331905676     169.775878052     254.663821169     509.327647246


     E_corr =    -0.064782001 [Ry] 


     rs =    10.000000000
     kf =     0.191915829

     qgrid:      [ndim = 8000 ]
               0.000009816       0.000019633       0.000029449       0.000039265       0.000049081
              50.935931627      63.669921896      84.893236829     127.339863424     254.679736663

     wgrid:      [ndim =16000 ]
               0.000004908       0.000009817       0.000014725       0.000019634       0.000024542
             101.865521596     127.331905676     169.775878052     254.663821169     509.327647246


     E_corr =    -0.061527674 [Ry] 

